tests/foreman/ui/test_errata.py

```diff
@@ -840,30 +830,47 @@ def test_positive_list_permission(
     :expectedresults: Check that the new user is able to see errata for one
         product only.
     """
-    module_org = module_org_with_parameter
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=function_sca_manifest_org.id,
+        product=PRDS['rhel8'],
+        repo=REPOS['rhst8']['name'],
+        reposet=REPOSET['rhst8'],
```

Comment:
Instead of using outdated rhst repos, why not we consider using rhsclient repos?
```suggestion
        repo=REPOS['rhsclient8']['name'],
        reposet=REPOSET['rhsclient8'],
```